package/com.unity.formats.usd/Runtime/Scripts/Behaviors/UsdAsset.cs

```diff
@@ -787,6 +787,11 @@ public string usdFullPath
     /// be destroyed and imported as a result.
     /// </summary>
     private void ApplyVariantSelectionState(Scene scene, UsdVariantSet variants) {
+      var primSource = variants.GetComponent<UsdPrimSource>();
```

Comment:
Question, the `GetComponent` method isn't something on `GameObject`? But `variants` is a `UsdVariantSet`... Is there something that makes USD object be somewhat compatible with Unity object with a simili shared interface? Some magic somewhere ^^

Reply:
This is a confusing naming convention -- UsdVariantSet in this case is a MonoBehaviour, so it too has a GetComponent method.

We should consider renaming everything in the Runtime/Scripts/Behaviors folder, it's currently very confusing :/